pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -275,8 +278,11 @@ func (i *CommandInitOption) Complete() error {
 	klog.Infof("karmada apiserver ip: %s", i.KarmadaAPIServerIP)
 
 	if !i.isExternalEtcdProvided() && i.EtcdStorageMode == "hostPath" && i.EtcdNodeSelectorLabels != "" {
-		if !i.isNodeExist(i.EtcdNodeSelectorLabels) {
-			return fmt.Errorf("no node found by label %s", i.EtcdNodeSelectorLabels)
+		labels := strings.Split(i.EtcdNodeSelectorLabels, ",")
+		for _, label := range labels {
+			if !i.isNodeExist(label) {
+				return fmt.Errorf("no node found by label %s", label)
+			}
```

Comment:
```suggestion
		if !i.isNodeExist(i.EtcdNodeSelectorLabels) {
			return fmt.Errorf("no node found by label %s", i.EtcdNodeSelectorLabels)
```

Here, we should verify if there are any nodes that match all the labels specified in a single string like "key1=value1,key2=value2", so we should not split the string into separate conditions.